docs/tutorials/helloworldparttwo.md

```diff
@@ -235,10 +289,19 @@ https://github.com/Unity-Technologies/com.unity.multiplayer.samples.poc/tree/fea
             ReadPermission = NetworkVariablePermission.Everyone
         });
 ```
+</details>
+
+### Introducing the concept of Ownership 
 
-We introduce the concept of ownership on a `NetworkVariable` (read and write permissions). For the purposes of this demo, the server will be authoritative on the `NetworkVariable` representing position. All clients are able to read the value, however.
+ In the `HelloWorldPlayer.cs `Script we  introduce the concept of ownership on a `NetworkVariable` (read and write permissions). For the purposes of this demo, the server will be authoritative on the `NetworkVariable` representing position. All clients are able to read the value, however.
```

Comment:
I just want to confirm that this is actual language on a `NetworkVariable`, @mattwalsh-unity. The concept of ownership, I thought, was reserved for `NetworkObject`s. I just don't want to conflate the two terms and confuse users. 

We could simplify things here and focus on just the permission of the `NetworkVariable` and how by defining the permissions as we have, only the server has permissions to write to the value of the `NetworkVariable`, while clients are only allowed to read from it.

Reply:
modifed text